lib/object_store/app/models/object_store/content_block.rb

```diff
@@ -0,0 +1,8 @@
+class ObjectStore::ContentBlock
```

Comment:
I think we should be good to inherit from the Edition model now, but I guess that can come later 👍